modules/ROOT/pages/test-ssl-enabled-sources-cookbook.adoc

```diff
@@ -0,0 +1,139 @@
+= Testing SSL-Enabled Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To test an application that has an HTTP Listener with SSL configured, you must configure an SSL truststore in your test to send authenticated requests to your code using a TLS certificate.
```

Comment:
I want to mention the two test writing methods and the Maven exception right in the beginning, so that we can focus on the primary text about using a TLS certificate. How about this:
```suggestion
Unless you are running your test from Maven, you create an MUnit test for an application that has an HTTP Listener with SSL configured by adding an SSL truststore in your test, to send authenticated requests to your code using a TLS certificate. You can either write your test, or you can scaffold your test using APIkit.
```